src/routes/userRoutes.test.js

```diff
@@ -191,4 +191,75 @@ describe('Testing deleteUser function', () => {
       expect(res.status.mock.calls[0][0]).toEqual(500)
       return expect(res.json.mock.calls[0][0].error.message).toEqual("Deleting user failed. Please try again")
   })
-})
\ No newline at end of file
+})
+
+describe('testing upDBPassword function', () => {
+  beforeEach(() => {
+       jest.clearAllMocks()
+  })
+
+  it('should send 400 error if user id not found', async() => { 
```

Comment:
Description should be `if input is invalid`

Reply:
I assume invalid input means the input format is not correct; however here is more about we can’t find the account to match the req body id... so account does not exist cakes more sense to me

Reply:
then your expect message should match the description text right? its good to make them consistent.